consensus/spos/worker.go

```diff
@@ -670,8 +686,16 @@ func (wrk *Worker) addBlockToPool(bodyBytes []byte) {
 	}
 }
 
-func (wrk *Worker) processReceivedHeaderMetric(cnsDta *consensus.Message) {
-	if wrk.consensusState.ConsensusGroup() == nil || !wrk.consensusState.IsNodeLeaderInCurrentRound(string(cnsDta.PubKey)) {
+func (wrk *Worker) processReceivedHeaderMetricForConsensusMessage(cnsDta *consensus.Message) {
+	if !wrk.consensusState.IsNodeLeaderInCurrentRound(string(cnsDta.PubKey)) {
```

Comment:
Necessary (just a question, for double-checking)?

Reply:
yes, for pre-activation only. after activation we don't have that info